tests/test_her.py

```diff
@@ -26,19 +25,23 @@ def test_import_error():
     assert "documentation" in str(excinfo.value)
 
 
+@pytest.mark.parametrize("n_envs", [1, 2])
 @pytest.mark.parametrize("model_class", [SAC, TD3, DDPG, DQN])
-@pytest.mark.parametrize("online_sampling", [True, False])
 @pytest.mark.parametrize("image_obs_space", [True, False])
-def test_her(model_class, online_sampling, image_obs_space):
+def test_her(n_envs, model_class, image_obs_space):
```

Comment:
why test multiple envs here if they are tested in `test_multiprocessing`?

Reply:
Since I set `n_envs=2` for almost all tests, I think I'd better remove `test_multiprocessing`. Do you agree?

Reply:
It does test the sub processes with it, I removed the `n_envs` from that test though.
For the rest I'm ok, even though I find it a bit weird that `n_envs=2` in all the other tests.